app/src/main/res/layout/fragment_category_product_list.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<layout xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <variable
+            name="category"
+            type="String" />
+    </data>
+
+    <androidx.appcompat.widget.LinearLayoutCompat xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:background="@color/background_color">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@={category}"
+            android:textSize="@dimen/dimen_categories_text_size"
+            android:textStyle="bold"
+            android:paddingStart="@dimen/dimen_category_name_padding_start"
+            android:paddingEnd="0dp"
+            android:paddingBottom="94dp"
```

Comment:
Please use value from `dimens.xml` instead of hardcoded one

Reply:
fixed